jenkins/run

```diff
@@ -1,20 +1,27 @@
 #!/bin/bash
 
-# Run commands within the pbench-devel-${IMAGE_KIND} container image.
-#
-# Generally you want to run a shell, and `source jenkins/python-setup.sh` to
-# initialize a Pbench testing environment.
+# Run commands within the pbench-${IMAGE_ROLE}-${IMAGE_KIND} container image.
 #
 # Several customizing inputs can be given via environment variables:
 #
 # EXTRA_PODMAN_SWITCHES []
 # IMAGE_REPO [quay.io/pbench]
+# IMAGE_ROLE [devel] (see Makefile for options)
 # IMAGE_KIND [fedora] (see Makefile for options)
+# IMAGE [image spec you want, overrides all of the above]
```

Comment:
This is starting to look like a general-purpose tool.  Should it be moved out of the `jenkins` directory and into, e.g., `utils`?